extensions/data-transfer/portability-data-transfer-generic/src/main/java/org/datatransferproject/datatransfer/generic/GenericTransferConstants.java

```diff
@@ -0,0 +1,6 @@
+package org.datatransferproject.datatransfer.generic;
+
+final class GenericTransferConstants {
+  public static final String SCHEMA_SOURCE_BASE =
+      "https://github.com/dtinit/data-transfer-project/blob/master";
```

Comment:
I am a little confused by this. 

It seems like we are using this only in the tests to compare what is getting serialized. If that is the case, we should compare what is currently at the runtime but not what is currently in the master branch. There is 100% chance that someone might be running a little older version of generic importers compared to master and that would break the tests. Instead, we should just read the file from classpath and do what we are doing.

Reply:
It gets used when building the schema source string to go in `GenericPayload`, in each `Serializer` class. That ends up in the payload to the importer endpoint.

It could go out of date, but I don't anticipate the code moving very often and would hope instances catch up relatively quickly. I did try to get it generated at runtime or compile time, but I couldn't find a way to get it to point to the right place (the path ended up in a build directory that doesn't exist in the repo) - if you know a way to do it that would be helpful!

One idea I had here was to switch to generating the POJOs from a JSON schema at build time, at which point the schema source could point to that file and make a bit more sense